src/log.js

```diff
@@ -469,6 +485,49 @@ export class Log {
       error.message = error.message.replace(USER_ERROR_SENTINEL, '');
     }
   }
+
+  /**
+   * Either redirects a pair of (errorId, ...args) to a URL where the full
+   * message is displayed, or displays it from a fetched table.
+   *
+   * This method is used by the output of the `transform-log-methods` babel
+   * plugin. It should not be used directly. Use the (*error|assert*|info|warn)
+   * methods instead.
+   *
+   * @param {string} id
+   * @param {...*} var_args
+   * @return {string}
+   */
+  expandLogMessage(id, var_args) {
```

Comment:
This is not getting DCE'd, which is why the bundle increased in size. Can comment out or remove simply.

Reply:
Plugin now handles multipass, so:

- method name gets minified
- method gets removed when not replacing arguments, so it should be DCE'd.